src/nodes/tahoma.ts

```diff
@@ -93,6 +93,13 @@ export = (RED: Red) => {
                     statusDoneText = 'Set to ' + msg.payload.position + '%';
                     expectedState = { open: true, position: msg.payload.position };
                     break;
+                case 'customRotation':
+                    commandName = 'rotation';
+                    parameters = [{ name: 'orientation', value: parseInt(msg.payload.orientation, 10) }];
+                    statusProgressText = 'Rotating to ' + msg.payload.orientation + 'degrees...';
+                    statusDoneText = 'Rotated to ' + msg.payload.orientation + '%';
```

Comment:
Hi @marekhalmo, thanks for the PR 😃!
Lines 99 and 100 use both a different "unit" (one is `degrees` and the other `%`). I just want to make sure this is intentional :)